src/content_script.js

```diff
@@ -70,22 +70,12 @@ function downloadThread({ as = Format.PNG } = {}) {
   const pixelRatio = window.devicePixelRatio;
   const minRatio = as === Format.PDF ? 2 : 2.5;
   window.devicePixelRatio = Math.max(pixelRatio, minRatio);
+
+  //Fix to the text shifting down when generating the canvas
+  document.body.style.lineHeight = '0.5';
```

Comment:
Good catch! Can you also restore it later? (near the `elements.restoreLocation()` call)?

Reply:
This should be done in the new commit.